core/src/main/java/org/web3j/tx/RawTransactionManager.java

```diff
@@ -107,7 +107,12 @@ public void setTxHashVerifier(TxHashVerifier txHashVerifier) {
 
     @Override
     public EthSendTransaction sendTransaction(
-            BigInteger gasPrice, BigInteger gasLimit, String to, String data, BigInteger value)
+            BigInteger gasPrice,
+            BigInteger gasLimit,
+            String to,
+            String data,
+            BigInteger value,
+            boolean constructor)
```

Comment:
this breaks backward compatibility

Reply:
Should we wait for release 5?

Reply:
release 5 will happen before devcon - if it is ok to wait then I suggest we do